src/app/sequence-editor/_sequence-editor-theme.scss

```diff
@@ -63,22 +63,21 @@
   .sequence-tab-right{
```

Comment:
Put a space between the selector (ex. `.sequence-tab-right`) and the opening curly brace "{"

Needs to be fixed in multiple places in this file